src/main/java/org/sobotics/guttenberg/commands/Update.java

```diff
@@ -0,0 +1,54 @@
+package org.sobotics.guttenberg.commands;
+
+import org.sobotics.guttenberg.clients.Updater;
+import org.sobotics.guttenberg.utils.CommandUtils;
+
+import fr.tunaki.stackoverflow.chat.Message;
+import fr.tunaki.stackoverflow.chat.Room;
+
+public class Update implements SpecialCommand {
+
+	private Message message;
+
+    public Update(Message message) {
+        this.message = message;
+    }
+	
+    @Override
+	public boolean validate() {
+		return CommandUtils.checkForCommand(message.getPlainContent(),"update");
+	}
+
+	@Override
+	public void execute(Room room) {
+		System.out.println("Load updater...");
+		Updater updater = new Updater();
+		System.out.println("Check for updates...");
+		
+		int update = updater.updateIfAvailable(); 
+		
+		if (update == -1) {
+			room.send("Automatic update failed!");
```

Comment:
It looks like it would be better to let `updateIfAvailable` throw an exception in case of a failed update, and let the thing calling the command catch it, so that for every command, you don't need to copy-paste this (and use magic values like 1 or -1), and have a standard `room.send("Automatic update failed!");` in the exception handler.